index.js

```diff
@@ -1,9 +1,27 @@
 var express = require('express');
+var fs = require('fs-extra');
+var retry = require('retry');
 var nconf = require('nconf');
 var bodyParser = require('body-parser');
+var storage = require('./storage/storage');
 
 nconf.argv().env().file('local.json');
 
+// We try to initialize the database 5 times
+fs.readFile('./storage/init_db.sql', 'utf8', function(err, script) {
+    if (err) throw err;
+    var operation = retry.operation();
+    operation.attempt(function () {
+        storage.execute(script, function(err) {
+            if (operation.retry(err)) {
+                console.log(err.message);
+                return ;
+            }
+            console.log('Database initialized');
+        });
+    });
+});
```

Comment:
yes :) and anyway the aim of this script __is__ to reinitialize the database by dropping everything inside